### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by orgads**

### CODE REVIEW

File: src/test/java/org/jenkinsci/plugins/sonargerrit/inspection/ReportDataChecker.java [hunk 1]
```diff
@@ -117,56 +113,56 @@ private static void checkReport1File(Report rep) {
         Assert.assertEquals("juice-events", c2.getPath());
 
         Component c3 = rep.getComponents().get(2);
-        Assert.assertEquals("com.megaproject.juice:juice-bootstrap", c2.getKey());
-        Assert.assertEquals("juice-bootstrap", c2.getPath());
+        Assert.assertEquals("com.megaproject.juice:juice-bootstrap", c3.getKey());
```
Comment (orgads) [inline src/test/java/org/jenkinsci/plugins/sonargerrit/inspection/ReportDataChecker.java]: Are these tests executed at all? All the tests passed even without this change, which is really strange...

**Comment by orgads** [comment]
PING!

File: src/test/java/org/jenkinsci/plugins/sonargerrit/inspection/ReportDataChecker.java (hunk 1 above)
Comment (aquarellian) [reply src/test/java/org/jenkinsci/plugins/sonargerrit/inspection/ReportDataChecker.java]: Method checkReport1File is not executed... I'll look into it, thanks.

**Comment by reda-alaoui** [comment]
A lot of refactoring that should fix most warning was done since. Please open a new PR with fixed conflicts if needed.

**Pull request closed without merging**